### ISSUE TITLE
Is IBM-400 and IBM on z/os supported

### ISSUE THREAD

**Issue by meghnad-globalids** [henryjw/knex-db2#19]

**Comment by henryjw** [comment]
Yup. That's actually my primary use case for this library. The only thing is that you have to install the appropriate driver on your client machine (the one using this lib).

**Comment by meghnad-globalids** [comment]
Any idea whether this v11.1.4fp4a_ntx64_odbc_cli driver version supports all three db2 types?
